server/app/views/applicant/ApplicantLayout.java

```diff
@@ -104,19 +106,28 @@ public Content render(HtmlBundle bundle) {
 
```

Comment:
The helper below is neccessary as there are 9 callers that still pass in an `Optional<String>` for the applicant name, coming from another `ApplicantService` method. It's out of scope for this PR to clean that up.